Makefile

```diff
@@ -53,3 +53,10 @@ update-with-container: build-runtime-image
 	$(RUNTIME) run -ti --rm -v $(PWD):/go/src/open-cluster-management.io/api:z -w /go/src/open-cluster-management.io/api $(RUNTIME_IMAGE_NAME) make update-scripts update-codegen-crds
 
 include ./test/integration-test.mk
+
+check-env:
+ifeq ($(GOPATH),)
+	$(warning "environment variable GOPATH is empty, auto set from go env GOPATH") 
+export GOPATH=$(shell go env GOPATH)
+endif
+.PHONY: check-env
```

Comment:
eof

Reply:
Should i add one more empty line after the L62?